tsconfig.base.json

```diff
@@ -23,6 +23,7 @@
         "packages/plugin-lighthouse/src/index.ts"
       ],
       "@quality-metrics/models": ["packages/models/src/index.ts"],
+      "@quality-metrics/models/testing": ["packages/models/test/index.ts"],
```

Comment:
I really like this separation :+1: 

Shouldn't we also exclude the `test` directory from `tsconfig.lib.json`?

Reply:
done